src/Management/src/Endpoint/ManagementWebHostBuilderExtensions.cs

```diff
@@ -310,18 +310,8 @@ public static IWebHostBuilder AddCloudFoundryActuator(this IWebHostBuilder hostB
         });
     }
 
-    /// <summary>
-    /// Adds Wavefront to the application.
-    /// </summary>
-    /// <param name="hostBuilder">
-    /// Your HostBuilder.
-    /// </param>
-    public static IWebHostBuilder AddWavefrontMetrics(this IWebHostBuilder hostBuilder)
-    {
-        return hostBuilder.ConfigureServices((context, collection) => collection.AddWavefrontMetrics());
-    }
-
     internal static void GetManagementUrl(this IWebHostBuilder webHostBuilder, out int? httpPort, out int? httpsPort)
+#pragma warning restore SA1612 // Element parameter documentation should match element parameters
```

Comment:
```suggestion
```

Reply:
Thanks, I addressed this along with some other changes.